libs/island-ui/utils/src/index.ts

```diff
@@ -1,3 +1,4 @@
 export { escapeGrid } from './lib/escapeGrid'
 export { deorphanize } from './lib/deorphanize'
 export { hexToRgba } from './lib/hexToRgba'
+export { TestSupport } from './lib/testSupport'
```

Comment:
Something similar to this might be needed to fix build errors
```suggestion
export type { TestSupport } from './lib/testSupport'
```